src/utils/generators/createTable.js

```diff
@@ -0,0 +1,11 @@
+export function createTable(rows, cols) {
+	const weekViewTable = document.querySelector(".week-view-table");
+	for (let col = 0; col < cols; col++) {
+		const tableCol = document.createElement("div");
+		weekViewTable.appendChild(tableCol).className = "table-column";
+		for (let row = 0; row < rows; row++) {
+			const tableCell = document.createElement("div");
+			tableCol.appendChild(tableCell).className = "table-cell";
```

Comment:
This line is a little bit confusing, you could separate this out in multiple steps.
I think you should:
1. Create a table cell;
2. Add the class name to it;
3. Append the cell to the `tableCol`.
```
const tableCell = document.createElement("div");
tableCell.className = "table-cell";
tableCol.appendChild(tableCell)
```